### PR TITLE
Sequence by master workflow

### PR DESCRIPTION
**Pull request by dzaslavskiy**
This is the first step in a refactor of the pipelines in this repo. This PR moves all test and prod deployment to a separate sequenced pipeline.

### CODE REVIEW

**Comment by folksgl** [comment]
Love the `cancel-deployment` step on failure. That should really help the dev cycle if things don't work the first time around.

**Review by folksgl — DISMISSED** [review]

File: .github/workflows/deploy-ALL-connectors-1.yaml [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+jobs:
+  base-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Wait for run-manifest in deploy ALL to finish
+        uses: lewagon/wait-on-check-action@v1.1.1
+        with:
+          ref: ${{ github.ref }}
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          check-regexp: (run-manifest|run-dbconfigs)
```
Comment (folksgl) [inline .github/workflows/deploy-ALL-connectors-1.yaml:23]: Can likely be shortened to
```suggestion
          check-regexp: run-(manifest|dbconfigs)
```

File: .github/workflows/deploy-ALL-connectors-0.yaml [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+jobs:
+  base-check:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Wait for run-manifest in deploy ALL to finish
+        uses: lewagon/wait-on-check-action@v1.1.1
+        with:
+          ref: ${{ github.ref }}
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          check-regexp: (run-manifest|run-dbconfigs)
```
Comment (folksgl) [inline .github/workflows/deploy-ALL-connectors-0.yaml:23]: See other comment on shorter regex

**Review by folksgl — CHANGES_REQUESTED** [review]
SK `.github/` folder is getting large/complex enough to likely deserve its own README. What's here now is hard enough for us to keep track of, not to mention anyone new to the project. Think you could add an overview for our actions/processes/workflows?

File: .github/workflows/deploy-ALL-connectors-1.yaml (hunk 1 above)
Comment (dzaslavskiy) [reply .github/workflows/deploy-ALL-connectors-1.yaml:23]: shorter maybe but I think as it currently is more readable

**Comment by dzaslavskiy** [comment]
Added a basic readme and long with smtp and s3.

**Review by folksgl — APPROVED** [review]

**Pull request merged**